From e34654e17fc968bf47266c83e374af70deac869a Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs3@bonedaddy.net>
Date: Wed, 23 Nov 2016 16:04:28 +0800
Subject: [PATCH] Uncomment SystemdService parameter in the dbus service file.
 (#74)

Without this the fwupd process will leak into the dbus cgroup.
---
 data/org.freedesktop.fwupd.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/org.freedesktop.fwupd.service.in b/data/org.freedesktop.fwupd.service.in
index 1d3046c..cde1e1d 100644
--- a/data/org.freedesktop.fwupd.service.in
+++ b/data/org.freedesktop.fwupd.service.in
@@ -3,4 +3,4 @@ Name=org.freedesktop.fwupd
 Documentation=http://www.fwupd.org/
 Exec=@servicedir@/fwupd/fwupd
 User=root
-#SystemdService=fwupd.service
+SystemdService=fwupd.service
-- 
2.7.4

